CRM/Mailing/BAO/Mailing.php

```diff
@@ -3173,10 +3173,11 @@ public static function getWorkflowFieldPerms() {
   }
 
   /**
-   * Whitelist of possible values for the entity_table field
+   * White-list of possible values for the entity_table field.
+   *
    * @return array
    */
-  public static function mailingGroupEntityTables($context = NULL) {
```

Comment:
Ah, I see. The function works kind of like a hook, in that the caller will pass in that variable whenever calling a `*EntityTables` function whether it's needed or not. In this case it's not needed. Fine to remove it or document it.